README.md

```diff
@@ -209,6 +209,19 @@ To build the Ruby code run:
 ```sh
 bazel build //rb/...
 ```
+
+To run tests on a browser:
+
+```sh
+bazel test --cache_test_results=no --test_output=all //rb:<browsername>-test 
+```
+
+To run a particular test on a browser:
+
+```sh
+bazel test --cache_test_results=no --test_output=all --test_arg="-e<Test_Target>" //rb:<browsername>-test
```

Comment:
Is there a list of Test Targets? How do I know what I can put here?

Reply:
`Test_Target` is not a list but a placeholder, just like `browsername`, for the specific test the user wants to run. You can see in line 224 I have given an example. 
`Example, Test_Target = Selenium::WebDriver::Devtools`

So if a contributor adds tests and wants to run it without running the whole test suite, he can put his test in `Test_Target` 

We can use a different name like `test_to_run`, or similar if you think it's better.

Reply:
Right, I'm looking for examples again, what else besides `Selenium::WebDriver::Devtools` works?

Reply:
We can have:
* Selenium::WebDriver::TakesScreenshot
* Selenium::WebDriver::Timeouts
* Selenium::WebDriver::Chrome::Driver
* Selenium::WebDriver::Firefox::Profile
* Selenium::Webdriver::Remote::Driver
etc...

Reply:
yes, listing a few examples is good. Easy to see what else can be used from that. thanks.

Reply:
Changes incorporated.